civihr_default_theme/scss/base/_mixins.scss

```diff
@@ -21,33 +21,29 @@
 }
 
 @mixin flex() {
-  display: -webkit-box;
```

Comment:
This should not be removed. The idea of creating this mixin is to have vendor prefixes for all browsers. You can disable the vendor prefix rule here.
```
/* stylelint-disable value-no-vendor-prefix */
@mixin flex() {
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
}
/* stylelint-enable value-no-vendor-prefix */
```